<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSRegisterForm</class>
 <widget class="QWidget" name="GBSRegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imgBanner">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>50</y>
     <width>346</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>212</y>
     <width>501</width>
     <height>534</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QLineEdit" name="leNickName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEmail">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVerifyCode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLoginPassword">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNickName">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>38</y>
      <width>29</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>128</y>
      <width>29</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVerifyCode">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>229</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>验证码</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginPassword">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>329</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>设置登陆密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegisterGBS">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>500</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: #1B2846;
font-size: 32px;
text-align: center;
</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAccountExisted">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblRegister">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>37</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblConstraintAndPrivacy">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>790</y>
     <width>240</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="chkbxAgree">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>790</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
